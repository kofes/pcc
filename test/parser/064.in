procedure bar(id: ^boolean = nil): integer;
begin
end;

begin
end
