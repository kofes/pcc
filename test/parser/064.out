Unexpected token ':' in pos (1, 34); expected '';'';
